### PR TITLE
AT_17.03_002 | Organization Folder

### PR DESCRIPTION
**Pull request by SawkaQA**
https://trello.com/c/vISVKfrJ/291-at1703002-organization-folder-delete-organization-folder

### CODE REVIEW

File: cypress/pages/createOrgFolder.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    this.createItem().click();
+    this.applyCreateItem().click();
+  }
+}
+
+export default Folder;
+
+
+
+
```
Comment (Rom-T) [inline cypress/pages/createOrgFolder.js:34]: we are still creating simple tests. Without Page objects. It and the format of its writing (how we will write it in our project) will be told later by our project manager.

File: cypress/e2e/DeleteOrgFolder.cy.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        it('Delete Organization Folder', () => {
+
+            // Create new Org Folder
+            const folder = new Folder();
+            folder.createNewFolder();
+
+            // Delete folderß
+            cy.contains(folderName).click()
+    
+            // cy.url().should('eq', `http://localhost:8080/job/${folderName}/`)
```
Comment (Rom-T) [inline cypress/e2e/DeleteOrgFolder.cy.js:18]: Our code should not have comments. Especially when it's commented out code. Delete please.

File: cypress/pages/createOrgFolder.js (hunk 1 above)
Comment (SawkaQA) [reply cypress/pages/createOrgFolder.js]: > we are still creating simple tests. Without Page objects. It and the format of its writing (how we will write it in our project) will be told later by our project manager.

I decided to try to implement this method. Now I will do it using the usual method, thank you

File: cypress/e2e/DeleteOrgFolder.cy.js (hunk 2 above)
Comment (SawkaQA) [reply cypress/e2e/DeleteOrgFolder.cy.js]: > Our code should not have comments. Especially when it's commented out code. Delete please.

To delete a folder, I need to create it first. I decided to write a comment that first of all I create a folder, then delete

**Review by Rom-T — APPROVED** [review]

File: cypress/e2e/DeleteOrgFolder.cy.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+/// <reference types="cypress"/>
+
+const folderName = 'Test';
+
+describe('<Organization Folder> Delete Organization Folder', () => {
```
Comment (Rom-T) [inline cypress/e2e/DeleteOrgFolder.cy.js:5]: Please also remove <> from the describe.

Comment (SawkaQA) [reply cypress/e2e/DeleteOrgFolder.cy.js]: Okay, thx

**Review by valentin-kushnarev — APPROVED** [review]

**Review by IrynaHryhoriv — APPROVED** [review]

**Pull request merged**